_posts/2017/06/2017-06-01-googles-magenta-kernel-hikey-960.md

```diff
@@ -58,7 +58,7 @@ With a recent update to the source, Google added initial support for the HiKey96
 
 And the instructions to get it running on the HiKey960 can be found here:
 
-[https://github.com/fuchsia-mirror/magenta/blob/master/docs/targets/hikey960.md]()
+[https://fuchsia.googlesource.com/zircon/+/master/docs/targets/hikey960.md]()
```

Comment:
The link target is still incorrect [this is a link]<!-- -->() produces a link to the current page, there is no magic to parse the link text to see if it happens to be a URI: [this is a link]()

Reply:
fixed